### ISSUE TITLE
Store learning rate in history when using `ReduceLROnPlateau`

### ISSUE THREAD

**Issue by BenjaminBossan** [skorch-dev/skorch#739]
Currently, we don't store the learning rate but it is now being exposed in PyTorch (we may need to check the versions though).

Reported by @jiajiexiao [here](https://github.com/skorch-dev/skorch/pull/738#discussion_r568267734).

**Comment by ParagEkbote** [comment]
I believe this issue has been fixed in #1075. WDYT?

cc: @BenjaminBossan

**Comment by BenjaminBossan** [comment]
Good find, thanks.
